aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -498,7 +498,6 @@ module aptos_token::token {
         token_data_id: TokenDataId,
         uri: String
     ) acquires Collections {
-        assert!(string::length(&uri) > 0, error::invalid_argument(EURI_TOO_SHORT));
```

Comment:
We don't do semantic validation. All the validation we do currently is to prevent getting too big string affecting indexer health. If we want to add the semantic validation, I think we can add them in batch to ensure consistency instead of only have this validation in this function which lead to inconsistent state.

Reply:
This is not semantic validation though. This enforces that uri is required and not empty.

Reply:
my point is that we don't want to enforce this for creators and only enforce this constraint for uri is not consistent with other fields